CHANGELOG.md

```diff
@@ -11,6 +11,13 @@
 * Adds launch.json option to suppress Just-In-Time compiler optimizations.
 * Due to the previous two items and work from the .NET Team, it is now possible to easily debug into ASP.NET itself in projects running against .NET Core 2.1 preview 1. Support for debugging into all the managed code in .NET Core will come in future .NET Core 2.1 builds. Instructions are in the [wiki](https://github.com/OmniSharp/omnisharp-vscode/wiki/Debugging-into-the-.NET-Framework-itself).
 
+#### Alternative Omnisharp Support
+
+* Adds support to use the "omnisharp.path" option to download alternative copies of OmniSharp. The possible values for this option are:
+    * "latest" - Use the latest CI build
+    * Version to use - The particular OmniSharp version the user desires like "1.29.2-beta.60"
+    * Some absolute path - The absolute path on disk that points to the executables (like Omnisharp.exe in Windows) or binaries to be used to start the OmniSharp server
```

Comment:
Copy whatever the option doc says for the path?

Reply:
Disregard in favor of Peter's comment.

Reply:
Some absolute path - the extension uses a local copy of OmniSharp. The value must point to a directory which contains OmniSharp, typically a user's build output directory for the OmniSharp-Roslyn project. Example: C:\\omnisharp-roslyn\\artifacts\\publish\\OmniSharp.Stdio\\win7-x64\\OmniSharp.exe.